src/integrators/integrator_utils.jl

```diff
@@ -40,6 +40,18 @@ function savevalues!(integrator::DiffEqBase.AbstractODEIntegrator{AlgType, IIP,
     saved, savedexactly
 end
 
+@inline function DiffEqBase.terminate!(integrator::DiffEqBase.AbstractODEIntegrator{AlgType,
```

Comment:
this is missing an argument.

Reply:
Oh, this was causing an issue with export from `OrdinaryDiffEq`. But then I made that as an extension of the `DiffEqBase` function.